aep/general/0151/aep.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 id: 151
 state: reviewing
+slug: lro
```

Comment:
This is a good short link...but is it a good slug/identifier?

Reply:
I can do a redirect from lro to a longer termed "long-running-operations"